htdocs/modulebuilder/index.php

```diff
@@ -1711,7 +1711,8 @@
 						$urltouse = dol_buildpath('/'.$regs[2].'/admin/'.$regs[1], 1);
 						$linktoenabledisable .= ' &nbsp; <a href="'.$urltouse.(preg_match('/\?/', $urltouse) ? '&' : '?').'save_lastsearch_values=1&backtopage='.urlencode($backtourl).'" title="'.$langs->trans("Setup").'">'.img_picto($langs->trans("Setup"), "setup", 'style="padding-right: 6px"').'</a>';
 					} else {
-						$urltouse = $urlpage;
+						//Case standard admin page
+						$urltouse = dol_buildpath(DOL_URL_ROOT.'/admin/'.$urlpage, 1);
```

Comment:
I think the problem is that in your descriptor, the config page was defined as "mypage" instead of "mypage@mymodule"